ui/README.md

```diff
@@ -189,17 +184,9 @@ Clicking the 'Raw' tab at the top of the UI displays the Json constructed and ha
 #### Now we will repeat the previous query but with a bigger time window - this time between 6AM and 8AM on 5/3/2005:
 - click on the 'M32:1' vertex
 - navigate to the 'query' page
-- click on the 'Get Elements' row
-- select 'RoadUse' on the view card
-- This time we are going to add a filter to the start and end times with a bigger time window
-- click 'Add Pre Aggregation filter'
-- Enter the following startDate filter:
-```
-property: startDate
-function: uk.gov.gchq.koryphe.impl.predicate.InDateRange
-start: 2005/05/03 07:00
-end: 2005/05/03 09:00
-```
+- select 'Get Elements' from the operation drop down
+- enter '05/03/2005' into the start and end date, but this time enter 06:00 and 08:00 as the start and end time.

```

Comment:
perhaps this should say:
- add a date range:
  - 05/03/2005 as the start date and add 06:00 in the adjacent box.
  - 05/03/2005 as the end date and add 08:00 in the adjacent box.